### PR TITLE
RHMAP-2628 - Adding Travis CI

### PR DESCRIPTION
**Pull request by jasonmadigan**

### CODE REVIEW

**Comment by jasonmadigan** [comment]
@wei-lee @corinnekrych adding some basic CI here (bigger task is to add tests to templates)

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+language: objective-c
+osx_image: xcode6.4
+xcode_project: FHStarterProject.xcodeproj
+xcode_sdk: iphonesimulator
+xcode_scheme: FHStarterProject
+
+before_install:
+  - gem install cocoapods --no-rdoc --no-ri --no-document --quiet
```
Comment (wei-lee) [inline .travis.yml]: it's not really using cocoapods here right?

**Comment by wei-lee** [comment]
looks good. At least we have something to make sure the app will be compiled. Next is to add some tests.

File: .travis.yml (hunk 1 above)
Comment (jasonmadigan) [inline .travis.yml]: @wei-lee nope, not yet. I'll remove it until later on.

**Comment by jasonmadigan** [comment]
@wei-lee exactly yeah. I'll merge now, we can extend on this later.

**Pull request merged**